test/get-workspaces-paths-test.js

```diff
@@ -166,7 +222,10 @@ describe(function() {
         fixturify.writeSync(tmpPath, {
           'packages': {
             'package-a': {
-              'package.json': stringifyJson({}),
+              'package.json': stringifyJson({
+                'name': 'package-a',
```

Comment:
These changes shouldn't be needed

Reply:
I checked out the file again from origin and made changes but somehow missed it - thanks